kategory/src/test/kotlin/kategory/generators/Generators.kt

```diff
@@ -8,6 +8,12 @@ inline fun <reified F, A> genApplicative(valueGen: Gen<A>, AP: Applicative<F> =
                     AP.pure(valueGen.generate())
         }
 
+inline fun <reified F, A> genEmpty(MKF: MonoidK<F>): Gen<HK<F, A>> =
```

Comment:
GenEmpty always returns the same value. Couldn't it be replaced by just using the monoid inline?